CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# Contributing to opentelemetry-ebpf-profiler
+
+The Go special interest group (SIG) meets regularly. See the
+OpenTelemetry
+[community](https://github.com/open-telemetry/community)
+repo for information on this and other SIGs.
+
+See the [public meeting
+notes](https://docs.google.com/document/d/19UqPPPlGE83N37MhS93uRlxsP1_wGxQ33Qv6CDHaEp0/edit#heading=h.4rdgawyis2hd)
+for a summary description of past meetings.
+
+## Development
+
+You can view and edit the source code by cloning this repository:
+
+```sh
+git clone https://github.com/open-telemetry/opentelemetry-ebpf-profiler.git
+```
+
+Run `make test` to run the tests instead of `go test`.
+
+Additionally, there is a `codespell` target that checks for common
+typos in the code. It is not run by default, but you can run it
+manually with `make codespell`.
+
+## Pull Requests
+
+### How to Send Pull Requests
+
+Everyone is welcome to contribute code to `opentelemetry-ebpf-profiler` via
+GitHub pull requests (PRs).
+
+To create a new PR, fork the project in GitHub and clone the upstream
+repo:
+
+```sh
+git clone https://github.com/open-telemetry/opentelemetry-ebpf-profiler
+```
+
+This would put the project in the `opentelemetry-go` directory in
+current working directory.
+
+Enter the newly created directory and add your fork as a new remote:
+
+```sh
+git remote add <YOUR_FORK> git@github.com:<YOUR_GITHUB_USERNAME>/opentelemetry-ebpf-profiler
+```
+
+Check out a new branch, make modifications, run linters and tests, and push the
+branch to your fork:
+
+```sh
+git checkout -b <YOUR_BRANCH_NAME>
+# edit files
+# update changelog
+git add -p
+git commit
+git push <YOUR_FORK> <YOUR_BRANCH_NAME>
+```
+
+Open a pull request against the main `opentelemetry-ebpf-profiler` repo.
+
+Avoid rebasing and force-pushing to your branch to facilitate reviewing the
+pull request.
+Rewriting Git history makes it difficult to keep track of iterations during
+code review.
+All pull requests are squashed to a single commit upon merge to `main`.
+
+### How to Receive Comments
+
+* If the PR is not ready for review, please put `[WIP]` in the title,
+	tag it as `work-in-progress`, or mark it as
+	[`draft`](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+* Make sure CLA is signed and CI is clear.
+
+### How to Get PRs Merged
+
+A PR is considered **ready to merge** when:
+
+* It has received two qualified approvals[^1].
+
+	This is not enforced through automation, but needs to be validated by the
+	maintainer merging.
+	* PRs introducing changes that have already been discussed and consensus
+		reached only need one qualified approval. The discussion and resolution
+		needs to be linked to the PR.
+	* Trivial changes[^2] only need one qualified approval.
```

Comment:
AFAIK we need two approvals for all PRs, so this should probably be reworded. 

CC: @athre0z

Reply:
Our experience is that opinions about what is trivial or not are different. To avoid friction and potential conflicts, we decided to always require 2 approvals. Especially for trivial changes, getting two approvals should not be an issue. We can also discuss this decision any time on Slack or in a profiling SIG meeting.

Reply:
I have removed trivial changes.